wagtail_transfer/static_src/components/ContentImportForm/index.js

```diff
@@ -68,16 +88,34 @@ export default function ImportContentForm({
     if (!sourcePage && destPage) {
       setDestPage(null);
     }
-  }, [source, sourcePage, destPage]);
+  }, [source, sourcePage, destPage, sourceModel, sourceModelObjectId]);
+
+  React.useEffect(() => {
+    if(sourceModel && !sourceModelObjectId) {
+      // There is a sourceModel but no sourceModelObjectId
+      const fetchNumPages = async () => {
+        const api = new ModelsAPI(source.page_chooser_api).query()
+        const page = await api.getModel(sourceModel.label)
```

Comment:
This line is a little confusing, looks like we're fetching model instances that match the model label of the `sourceModel` instance?

If that's correct, could we rename `sourceMode.labell` to `sourceInstance.model_label` and `api.getModel` to `api.getModelInstances`?